pkg/gpu/nvidia/metrics/metrics.go

```diff
@@ -179,6 +205,23 @@ func (m *MetricServer) updateMetrics(containerDevices map[ContainerID][]string)
 			MemoryUsed.WithLabelValues(container.namespace, container.pod, container.container, "nvidia", d.UUID, *d.Model).Set(float64(*mem.Global.Used) * 1024 * 1024) // memory reported in bytes
 		}
 	}
+	node_name := os.Getenv("NODE_NAME")
```

Comment:
nit: use `nodeName` may be more align with Go style: go/go-style/guide#mixed-caps

Reply:
By the way, are you planning to add a downward API env NODE_NAME to device plugin yaml? If you are, it would be nice if you can also update: https://github.com/GoogleCloudPlatform/container-engine-accelerators/blob/master/cmd/nvidia_gpu/device-plugin.yaml with that env as a reference.

Reply:
Yes, my bad. Changed.